### ISSUE TITLE
您好，可以问一下是直接下载cpt的模型进行推理就可以了吗？如果不是，Model里面是自己训练出来的权重吗？

### ISSUE THREAD

**Issue by haaddii** [Zaaachary/QuestionGeneration#1]
如题……

**Comment by zephyrli-coding** [comment]
是的  可以直接加载CPT的模型，具体的可以在huggingface上下载。https://huggingface.co/fnlp/cpt-large
CPT的详情请见他们的论文CPT: A Pre-Trained Unbalanced Transformer for Both Chinese Language Understanding and Generation
